app/src/main/java/org/mozilla/fenix/FenixApplication.kt

```diff
@@ -253,6 +255,14 @@ open class FenixApplication : LocaleAwareApplication() {
         }
     }
 
+    private fun warmBrowsersCache() {
+        // We avoid blocking the main thread for BrowsersCache on startup by loading it on
+        // background thread.
```

Comment:
Only real nit: We should say that "We try to avoid blocking" because this warmup method could still be running by the time that the app gets to first time that the `BrowsersCache` is needed. Not sure it's necessary to point that out, but perhaps it is? Up to you!